src/leaflike/bookmarks/db.clj

```diff
@@ -24,23 +30,34 @@
                             (helpers/where [:= :member_id member_id])
                             sql/format)))
 
+(defn- build-where-clause
+  [{:keys [member_id tag]}]
+  (let [where-clause [:and [:= :member_id member_id]]
+        where-clause (if tag
```

Comment:
This `if` can be replaced by a `when`

Reply:
@trycatcher : Using `when` here would cause `where-clause` to be `nil` when `tag` is falsey.